examples/ida_binaryai.py

```diff
@@ -3,25 +3,31 @@
 import json
 import idaapi
 import idautils
+import idc
+import ida_kernwin
+from ida_netnode import Netnode
```

Comment:
It's not a perfect way. as:
1. the package name 'ida_netnode' is conflicted with the official IDAPython. (tested in IDA7.5 macOS version)
2. as a standalone plugin, import a third-party package is not user friendly, is it possible to re-implement the same feature directly.